### ISSUE TITLE
[Category Details Screen] Labels On Disabled Buttons Should Be Grayed Out

### ISSUE THREAD

**Issue by rhsalas3** [willowtreeapps/eyespeak-ios#531]
### **Description**:
Currently on the Category Details screen, for a button that is disabled, the background of the button is properly grayed out but the label (and switch for the 'Show Category' button) are not. This makes it a bit more difficult for users to tell the difference between a button that is tappable and one that is disabled. We should update the buttons to look more like designs, or at least how they were prior to the screen's layout changes.

This issue ticket is related to #485 and is caused by changes made in PR #527 . Notes about the expected behavior are in the PR conversation, and original AC that define expected behavior of the buttons for specific categories can be found in the conversation of issue ticket #465 .

**Acceptance Criteria**:
- [x] Labels on disabled buttons of the Category Details screen should be grayed out.
- [x] Categories with disabled buttons: 
    - [x] For 'My Sayings' -> the 'Remove Category', and 'Show Category' buttons should be disabled
    - [x] For 'Recents' -> the 'Edit Phrases' button should be disabled
    - [x] For '123' -> the 'Edit Phrases' button should be disabled

### **Design**:
Example of a disabled button ('Remove Category') from the Category Details screen.
https://www.figma.com/file/mNrwUygVhTmuWnKNHHkenR/Vocable---Current?node-id=3641%3A27468

**Screenshot(s)**:

![CategoryDetailsScreen_disabledButtons_notObviouslyGrayedOut_comparedToPreviousVersion](https://user-images.githubusercontent.com/34660497/161273804-2c217f36-ba14-4c3f-a15a-8cfcdabe9f87.png)

**Comment by rhsalas3** [comment]
Tested on PR branch and this issue has been fixed.

PR #536
